iaso/serializers.py

```diff
@@ -22,6 +22,12 @@ def validate(self, attrs):
             request = self.context.get("request")
             if request:
                 app_id = request.query_params.get("app_id", None)
+                # For setuper, account name is same as app id for Main Project, so get the first user account with super user status linked to current super user multi account
+                if app_id is None:
+                    account_tenant_user = (
+                        self.user.tenant_users.filter(account_user__is_superuser=True).first().account_user
+                    )
+                    app_id = account_tenant_user
```

Comment:
We really should not depend on the coincidence that `str(account_tenant_user) == app_id`.

Thinking about it a bit more, how about we return a token for the actual user if the app_id is None?
What if we changed
```python
if self.user.tenant_users.exists():
```
for
```python
if app_id and self.user.tenant_users.exists():
```
?

In that case we could simply say "if you don't specify an app_id, we'll give you a token for the main user".